index.js

```diff
@@ -1,4 +1,4 @@
-const honkify = () => {
+const honkify = (elementToHonkify = document) => {
```

Comment:
This will hit SSR issues since `document` won't exist during builds.

If we default to `false` we can work around that, though.

```suggestion
const honkify = (elementToHonkify = false) => {
```

Reply:
ah, mk, good call. Thanks for the commit suggestions : D